### PR TITLE
Feature item filters

### PR DESCRIPTION
**Pull request by d-mooers**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ const App = () => {
     username: "",
     email: "",
     password: "",
-    _id: "609cb2b88951a63fccdcf790",
-    authId: "db3edf52-df87-45b9-9ea5-cd3a38cc2807",
+    _id: "60a339a6417c58df0f213c81",
+    authId: "1a38fffa-1e67-4c1f-8be5-12964c824174",
     loggedIn: false,
```
Comment (slaterOwen) [inline client/src/App.js:24]: Was there a reason the default ids where changed?

**Review by slaterOwen — COMMENTED** [review]
Nice additions to the code. You used comments throughout the code which was nice to see. Everything looks good to me.

File: client/src/App.js (hunk 1 above)
Comment (d-mooers) [reply client/src/App.js:24]: I was working on this while Chase was going through to update the Login page, so I had to refresh the access key a couple times so I could test stuff out.  Each login generates a new authId for that session

File: client/src/components/RegisterPage/Registration.js [hunk 2]
```diff
@@ -50,16 +55,24 @@ const Register = (props) => {
...
-      window.location.href = "/browse";
+      try {
+        const resp = await axios.post("http://127.0.0.1:5000/users", {
+          ...user,
+          password: sha256(user.password),
+        });
+        setUser(resp.data);
+        props.history.push("/profile");
+      } catch (e) {
+        console.log(e);
```
Comment (ChaseVZ) [inline client/src/components/RegisterPage/Registration.js:74]: This part was in need of an update. Good stuff.

File: client/src/components/RegisterPage/Registration.js [hunk 3]
```diff
@@ -10,38 +13,40 @@ const styles = makeStyles((theme) => ({
...
         email: user["email"],
         password: value,
         secondPass: user["secondPass"],
       });
     } else if (name === "secondPass") {
-      setUser({
+      setUserInfo({
         username: user["username"],
         email: user["email"],
         password: user["password"],
```
Comment (ChaseVZ) [inline client/src/components/RegisterPage/Registration.js:52]: could still use '...user' but works

File: client/src/components/ViewItems/Item.js [hunk 4]
```diff
@@ -8,42 +8,78 @@ import CardMedia from "@material-ui/core/CardMedia";
...
+    <Grid item xl={4} lg={4} md={4} sm={5}>
+      <div className={classes.grid}>
+        <Card
+          className={classes.root}
+          onMouseEnter={() => setHover(true)}
+          onMouseLeave={() => setHover(false)}
+        >
+          <CardActionArea onClick={handleClick}>
+            <CardMedia
+              className={hover ? classes.mediaLarge : classes.mediaSmall}
```
Comment (ChaseVZ) [inline client/src/components/ViewItems/Item.js:63]: I like it

**Review by ChaseVZ — COMMENTED** [review]
Looks good!

File: client/src/components/RegisterPage/Registration.js (hunk 3 above)
Comment (d-mooers) [reply client/src/components/RegisterPage/Registration.js:52]: Agreed, when I did this I just changed the name of the function and left everything else as is though.

**Pull request merged**